link_test.go

```diff
@@ -1883,6 +1883,53 @@ func TestLinkSet(t *testing.T) {
 	if link.Attrs().Group != 42 {
 		t.Fatal("Link group not changed")
 	}
+
+	err = LinkAddAltName(link, "altname")
```

Comment:
any chance to move this to a separate test ?

feels like this tests is already testing quite alot.